README.md

```diff
@@ -12,12 +12,10 @@ The following table compares the two converter engines:
 | ----- | ----- | ----- |
 | **Template language** | [Handlebars](https://handlebarsjs.com/) | [Liquid](https://shopify.github.io/liquid/) |
 | **Template authoring tool** | Self-hosted web-app | [VS Code extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-health-fhir-converter)|
-| **Supported conversions** | 1. HL7v2 to FHIR <br> 2. CCDA to FHIR | 1. HL7 v2 to FHIR <br> 2. *CCDA to FHIR** |
-| **Available as** | 1. Self-deployed web service <br> (on-prem or on Azure)| 1. Command line tool <br> 2. $data-convert operation of Azure API for FHIR <br> 3. *$data-convert operation on  FHIR Server for Azure**|
+| **Supported conversions** | 1. HL7v2 to FHIR <br> 2. CCDA to FHIR | 1. HL7 v2 to FHIR <br> 2. CCDA to FHIR (to be released soon)|
+| **Available as** | 1. Self-deployed web service <br> (on-prem or on Azure)| 1. Command line tool <br> 2. $data-convert operation in Azure API for FHIR <br> 3. $data-convert operation in  FHIR Server for Azure (to be released soon)|
```

Comment:
Hey @ranvijaykumar, I suspect the to be released soon is on the incorrect line here? It's my understanding that the conversion is available in the FHIR Server for Azure and Not the Azure API for FHIR at the moment. Is that correct?

Reply:
Thanks for catching this :)